### PR TITLE
added pexels to code-playground

### PR DESCRIPTION
**Pull request by TatuSilva**

### CODE REVIEW

File: code-playgrounds.md [hunk 1]
```diff
@@ -23,3 +23,4 @@ Code Playground | Description
 [Trinket.io](https://trinket.io/) | Trinket is a website that allows you to create and test code without creating an account. Mainly it is used for python but they have other languages as well. **Features(examples, plans, tutorials)**
 [Postdrop.io](https://app.postdrop.io) | Postdrop. A simple tool to help developers with HTML email.
 [StackBlitz](https://stackblitz.com) | StackBlitz. A nice playground to create, edit & deploy fullstack apps. It supports JavaScript frameworks such as Angular, React, Ionic, TypeScript, Svelte, and RxJs.
+[Pexels](https://www.pexels.com) | Pexels. Free stock images, photos and videos.
```
Comment (lucasgdb) [inline code-playgrounds.md:26]: is this a code playground?

**Review by lucasgdb — CHANGES_REQUESTED** [review]
Please, move the link to the web section in README

File: code-playgrounds.md (hunk 1 above)
Comment (TatuSilva) [reply code-playgrounds.md:26]: sorry! first timer here. hope to get it right this time.

**Review by lucasgdb — APPROVED** [review]

File: code-playgrounds.md (hunk 1 above)
Comment (lucasgdb) [reply code-playgrounds.md]: right :)

**Pull request merged**